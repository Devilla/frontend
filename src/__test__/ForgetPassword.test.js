// import React from 'react';
// import { shallow, mount } from 'enzyme';
// import renderer from 'react-test-renderer'
// import configureStore from 'redux-mock-store'
// import { Provider } from 'react-redux'
// import {createStore} from 'redux'
// import { load, loaded } from 'ducks/loading';
// import { ForgetPassword } from 'components';
// import { store } from 'App';
//
// // Snapshot for Home React Component
// describe('>>> ForgetPassword --- Snapshot',()=>{
//     it('+++capturing Snapshot of ForgetPassword', () => {
//         const renderedValue =  renderer.create(<ForgetPassword store={store} />).toJSON()
//         expect(renderedValue).toMatchSnapshot();
//     });
// });
