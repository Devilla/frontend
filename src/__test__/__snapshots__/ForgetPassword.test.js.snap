// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`>>> ForgetPassword --- Snapshot +++capturing Snapshot of ForgetPassword 1`] = `
<div
  className="forgetpassword-container"
>
  <div
    className="authpage section innerpage"
  >
    <div
      className="container"
    >
      <div
        className="wrapper"
      >
        <div
          className="animated fadeIn center"
          style={
            Object {
              "animationDelay": "0ms",
              "pointerEvents": "all",
            }
          }
        >
          <form
            className="loginfrm"
            data-name="Login Form"
            method="POST"
            onSubmit={[Function]}
          >
            <h3
              className="dashed pt-5 h2 text-center"
            >
              Forgot your password
            </h3>
            <div
              className="section-divider-line"
            />
            <div
              className="row justify-content-center"
            >
              <div
                className="frmcntl pb-2 col-md-5"
              >
                <div>
                  <h3
                    className="pb-3 lead"
                  >
                    Enter your email address below and well send you a link to reset your password.
                  </h3>
                  <div
                    className="frmcntl pb-4"
                  >
                    <input
                      className="field w-input"
                      id="email"
                      name="email"
                      onBlur={[Function]}
                      onChange={[Function]}
                      placeholder="Email Address"
                      type="email"
                      value=""
                    />
                    <span
                      className="help-block"
                    >
                      <p
                        className="website-error"
                      >
                        
                      </p>
                    </span>
                  </div>
                </div>
                <div
                  className="frmcntl pb-4"
                >
                  <input
                    className="btn btn-primary "
                    onClick=""
                    type="submit"
                    value="Send reset password email"
                  />
                </div>
              </div>
            </div>
          </form>
          <span
            className="row pt-5 support "
          >
            <span
              className="h5 lead"
            >
              Trouble logging in? 
              <a
                onClick={[Function]}
                style={Object {}}
              >
                Contact Support
              </a>
            </span>
          </span>
        </div>
      </div>
    </div>
  </div>
</div>
`;
